mama/c_cpp/src/c/datetime.c

```diff
@@ -633,7 +646,7 @@ mamaDateTime_addSeconds(mamaDateTime dateTime,
 
         mamaDateTime_addWholeSeconds (dateTime, wholeSeconds);
         mamaDateTime_addMicroseconds (dateTime, microseconds);
-        mamaDateTimeImpl_setHasTime            (*dateTime);
+        mamaDateTimeImpl_setHasTime            ((mama_time_t*)dateTime);
```

Comment:
I know it was consistent with before but could you fix this indentation please while you're in this area?